VERSION ""

NS_ :
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_

BS_:

BU_:

BO_ 100 ActualVehicleSpeed: 8 Vector__XXX
 SG_ ActualVehicleSpeed : 0|24@1+ (0.5,0.0) [0|320] "kph" Vector__XXX

BO_ 202 ChargeLevel: 8 Vector__XXX
 SG_ ChargeLevel : 0|24@1+ (0.25,-100.0) [-100|1000] "%" Vector__XXX

BO_ 500 GreenLed: 8 Vector__XXX
  SG_ GreenLed : 0|24@1+ (1,0.0) [0|255] "status" Vector__XXX

BO_ 604 VehicleStatus: 8 Vector__XXX
  SG_ VehicleStatus : 0|24@1+ (1,0.0) [0|1000] "" Vector__XXX

BO_ 605 BatteryPackVoltage: 8 Vector__XXX
  SG_ BatteryPackVoltage : 0|24@1+ (1,0.0) [0|10000] "v" Vector__XXX

BO_ 502 RedLed: 8 Vector__XXX
  SG_ RedLed : 0|24@1+ (1,0) [0|255] "status" Vector__XXX


BA_DEF_ SG_  "SignalType" STRING ;
BA_DEF_ SG_  "SignalLongName" STRING ;
BA_DEF_ BO_  "GenMsgCycleTime" INT 0 10000;
BA_DEF_DEF_  "SignalType" "";
BA_DEF_DEF_  "SignalLongName" "";
BA_DEF_DEF_  "GenMsgCycleTime" 50;

BA_ "GenMsgCycleTime" BO_ 100 50;
BA_ "GenMsgCycleTime" BO_ 202 50;
BA_ "GenMsgCycleTime" BO_ 500 50;
BA_ "GenMsgCycleTime" BO_ 502 50;
BA_ "GenMsgCycleTime" BO_ 604 50;
BA_ "GenMsgCycleTime" BO_ 605 50;

